### PR TITLE
Username and Password added

### PR DESCRIPTION
**Pull request by Himanshi-Khandelwal**
Modified Home.md

### CODE REVIEW

File: 00-Home.md [hunk 1]
```diff
@@ -58,6 +58,13 @@ python main.py
 
 The demo should be up and running on `http://localhost:8080/serverapi/`
 
+The username and password required for various API endpoints is:
```
[the remark this answers is not in the document]
Comment (Himanshi-Khandelwal) [reply 00-Home.md]: oks, I will do this.

**Comment by Himanshi-Khandelwal** [comment]
@xadahiya Please review.

**Comment by chrizandr** [comment]
@xadahiya Please review and merge if it is okay.

File: 00-Home.md [hunk 2]
```diff
@@ -58,6 +58,19 @@ python main.py
...
 The demo should be up and running on `http://localhost:8080/serverapi/`
 
+The default credentials for API endpoints that require authentication are
+```bash
+username : 1
+password : test
+```
+
+
+To add authorised users to your API:
```
Comment (xadahiya) [inline 00-Home.md]: You should also specify where new users can be added like with CLI arguments and main.py

**Comment by Mec-iS** [comment]
I think username and password have been deprecated by the new Token-based system. See [Auth](https://github.com/HTTP-APIs/hydra-ecosystem-wiki/blob/master/Auth.md)

**Pull request closed without merging**